packages/rs-json-schema-compatibility-validator/src/rules/rule_set.rs

```diff
@@ -0,0 +1,1378 @@
+                        )
+                            .into(),
+                        (
+                            json!({ "contentMediaType": "application/x.dash.dpp.identifier" }),
+                            json!({}),
+                            Some(JsonSchemaChange::Remove(RemoveOperation {
+                                path: "/contentMediaType".to_string(),
+                            })),
+                        )
+                            .into(),
+                        (
+                            json!({ "contentMediaType": "application/x.dash.dpp.identifier" }),
+                            json!({ "contentMediaType": "application/unknown" }),
+                            Some(JsonSchemaChange::Replace(ReplaceOperation {
+                                path: "/contentMediaType".to_string(),
+                                value: json!("application/unknown"),
+                            })),
+                        )
+                            .into(),
+                    ],
+                },
+            ),
+            (
+                "byteArray",
+                CompatibilityRules {
+                    allow_addition: false,
+                    allow_removal: false,
+                    allow_replacement_callback: FALSE_CALLBACK.clone(),
+                    subschema_levels_depth: None,
+                    inner: None,
+                    #[cfg(any(test, feature = "examples"))]
+                    examples: vec![
+                        (
+                            json!({}),
+                            json!({ "byteArray": true }),
+                            Some(JsonSchemaChange::Add(AddOperation {
+                                path: "/byteArray".to_string(),
+                                value: json!(true),
+                            })),
+                        )
+                            .into(),
+                        (
+                            json!({ "byteArray": true }),
+                            json!({}),
+                            Some(JsonSchemaChange::Remove(RemoveOperation {
+                                path: "/byteArray".to_string(),
+                            })),
+                        )
+                            .into(),
+                        (
+                            json!({ "byteArray": true }),
+                            json!({ "byteArray": false }),
+                            Some(JsonSchemaChange::Replace(ReplaceOperation {
+                                path: "/byteArray".to_string(),
+                                value: json!(false),
+                            })),
+                        )
+                            .into(),
+                    ],
+                },
+            ),
+            (
+                "prefixItems",
+                CompatibilityRules {
+                    allow_addition: false,
+                    allow_removal: false,
+                    allow_replacement_callback: None,
```

Comment:
Why None?

Reply:
You can't replace prefixItems because it's contains an object. You can replace object to other types but json schema doesn't allow other types for this keyword. If you replace element inside the object that it's inner structure that handled by inner rules.

Reply:
Okay